docs/core/entity/light.md

```diff
@@ -11,27 +11,51 @@ A light entity controls the brightness, hue and saturation color value, white va
 | Name | Type | Default | Description
 | ---- | ---- | ---- | ----
 | brightness | int | None | Return the brightness of this light between 0..255
+| color_mode | string | None | Return the color mode of the light. Must be present in the `supported_color_modes` property.
 | color_temp | int | None | Return the CT color value in mireds.
 | effect | String | None | Return the current effect.
 | effect_list | list | None | Return the list of supported effects.
 | hs_color | list | None | Return the hue and saturation color value [float, float].
 | is_on    | bool | bool  | Returns if the light entity is on or not.  
 | max_mireds | int | int | Return the warmest color_temp that this light supports.
 | min_mireds | int | int | Return the coldest color_temp that this light supports.
+| rgb_color | list | None | Return the rgb color value [int, int, int].
+| rgbw_color | list | None | Return the rgbw color value [int, int, int, int].
+| rgbww_color | list | None | Return the rgbww color value [int, int, int, int, int].
+| supported_color_modes | list | None | Flag supported color modes.
 | supported_features | int | int | Flag supported features.
-| white_value | int | None | Return the white value of this light between 0..255.
+| white_value | int | None | Return the white value of this light between 0..255. This is deprecated and will be removed in Home Assistant 2021.10.
+| xy_color | list | None | Return the xy color value [float, float].
+
+## Color Modes
+
+| Constant | Description
+|----------|-----------------------
+| `COLOR_MODE_UNKNOWN` | 
+| `COLOR_MODE_ONOFF` | The light can be turned on or off. This mode must be the only supported mode if supported by the light.
+| `COLOR_MODE_BRIGHTNESS` | The light can be dimmed. This mode must be the only supported mode if supported by the light.
+| `COLOR_MODE_COLOR_TEMP` | The light can be dimmed and its color temperature is present in the state.
+| `COLOR_MODE_HS` | The light can be dimmed and its color is present in the state as an (h, s) tuple (no brightness).
+| `COLOR_MODE_RGB` | The light can be dimmed and its color is present in the state as an (r, g, b) tuple (not normalized for brightness).
```

Comment:
In the property we say that it's a list. Here we say tuple. Are we talking about different things or should it be the same?

Reply:
Thanks Martin, it should be tuples in both cases.